Userland/cal.cpp

```diff
@@ -0,0 +1,46 @@
+#include <stdio.h>
+#include <time.h>
+
+int dayOfWeek(int day, int month, int year)
```

Comment:
Serenity uses `lower_case_with_underscores` for function and variable names, so `day_of_week()` here.